### ISSUE TITLE
Consider add support for colorized-outputs in the screenshot

### ISSUE THREAD

**Issue by Dentrax** [asciinema/asciinema#409]
I don't know exactly either this is bug or lack of a feature that colorize-output in the screenshot.

**Screenshosts:**

How it is supposed to look like in the screenshot: _(What I've been expecting, actually)_

![screenshot2](https://user-images.githubusercontent.com/16493751/87972594-d8a5b180-cacf-11ea-8a44-16e020f8b232.png)

Actual:

![screenshot1](https://user-images.githubusercontent.com/16493751/87972492-a4ca8c00-cacf-11ea-9598-21e014075db7.png)

Link: https://asciinema.org/a/348527 _(...at time 00:10)_

**Comment by sickill** [comment]
Good catch. It seems like the generated SVG is lacking the colors for some of the text.
